figure/main.py

```diff
@@ -47,7 +25,7 @@ def update_legends(p, q_list, hover):
 def get_plot(inp_x, inp_y, inp_clr):
     """Returns a Bokeh plot of the input values, and a message with the number of COFs found."""
     q_list = [config.quantities[label] for label in [inp_x, inp_y, inp_clr]]
-    results_wnone = get_data_aiida(q_list)  #returns [inp_x_value, inp_y_value, inp_clr_value, group.label]
+    results_wnone = get_data_aiida(q_list)  #returns ***
```

Comment:
I don't think *** is clearer here...
let's remove this comment and instead document in the docstring of `get_data_aiida` what the function returns